swagger-current.yml

```diff
@@ -1,13 +1,14 @@
 openapi: 3.0.0
 info:
   title: ONS Contact Centre API
-  version: "5.10.17-oas3"
+  version: "5.11.18-oas3"
```

Comment:
interesting that the versions got switched from 5.10.xx to 5.11.xx  . Wonder if that. was intentional or by accident ? Might be worth double checking.

Reply:
I took this as being correct. Mentioned to Phil yesterday but he didn't seem interested